symengine/series_generic.cpp

```diff
@@ -1,156 +1,219 @@
-#include <exception>
-#include <algorithm>
-#include <iterator>
 #include <symengine/series_generic.h>
-#include <symengine/dict.h>
+#include <symengine/series_visitor.h>
+#include <symengine/derivative.cpp>
```

Comment:
You don't need to include this here.
Multiple symbols error is because of this.